src/commonmark/en/content/developer/web-api/metadata.md

```diff
@@ -3947,16 +3947,29 @@ The contents of file resources are not directly accessible but are
 referenced from other objects (such as data values) to store binary
 content of virtually unlimited size.
 
-Creation of the file resource itself is done through the `/api/fileResources` endpoint as a multipart upload POST-request:
+Creation of the file resource itself is done through either `/api/fileResources`
```

Comment:
I think we can be more specific up front about the difference between the two endpoints, and when you would want to use each one. Perhaps something like:

To create a file resource itself, use the endpoint `/api/fileResources`:

```bash
curl "https://server/api/fileResources" -X POST
  -F "file=@/path/to/file/name-of-file.png"
```

If you do this, you must afterwards reference (assign) the file resource from another object.

To create a file resource, and at the same time create a data value that references it, use the `/api/dataValues/file` endpoint:

```bash
curl "https://server/api/dataValues/file?de=xPTAT98T2Jd
  &pe=201301&ou=DiszpKrYNg8&co=Prlt0C1RF0s" -X POST
  -F "file=@/path/to/file/name-of-file.png"
```

For the `api/fileResources` endpoint ...